Domain/Models/Booking.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AirportTicketBookingSystem.Domain.Models
+{
+    public class Booking
+    {
+        public int Id { get; set; }
+        public Flight Flight { get; set; }
+        public Passenger Passenger { get; set; }
+        public SeatClass? SeatClass { get; set; }
```

Comment:
You don't have to make it nullable, usually with enums we add a default value called for example `None = 0` and just use that instead. Better than dealing with nulls.